data/en/ecommerce_0001.json

```diff
@@ -0,0 +1,3481 @@
+                                            "type": "Percentage",
+                                            "label": "Sales to other businesses (B2B) and public authorities (B2G)",
+                                            "description": "",
+                                            "min_value": {
+                                                "value": 0,
+                                                "exclusive": false
+                                            },
+                                            "max_value": {
+                                                "value": 100,
+                                                "exclusive": false
+                                            },
+                                            "decimal_places": 1
+                                        }
+                                    ],
+                                    "calculations": [
+                                        {
+                                            "calculation_type": "sum",
+                                            "answers_to_calculate": [
+                                                "answer05718a5c-7a65-4775-b28f-e2787cfa6ac1",
+                                                "answer2565ef7d-9fda-4485-b05c-1e4678fa97cd"
+                                            ],
+                                            "conditions": ["equals"],
+                                            "value": 100
+                                        }
+                                    ]
+                                }
+                            ]
+                        },
+                        {
+                            "id": "block1400",
+                            "type": "Question",
+                            "questions": [
+                                {
+                                    "id": "question1400",
+                                    "title": "During {{ metadata['period_str'] }}, did {{ metadata['trad_as_or_runame'] }} receive any orders for goods or services via your business&apos;s own website or &apos;app&apos;, or e-commerce market places?",
+                                    "guidance": {
+                                        "content": [
+                                            {
+                                                "description": "<strong>Your business&apos;s own website or &apos;app&apos; should include:</strong>"
+                                            },
+                                            {
+                                                "list": ["parent or affiliate businesses", "extranets"]
+                                            }
+                                        ]
+                                    },
+                                    "definitions": [
+                                        {
+                                            "title": "What is an e-commerce market 
```

Comment:
I think this q_code should be 521 if I am interpreting the document correctly. 512 is also used in another place. Same for both files.